apis/python/src/tiledbsoma/options/_tiledb_create_options.py

```diff
@@ -168,7 +173,7 @@ def from_platform_config(
         """
         create_entry = _dig_platform_config(platform_config, cls, ("tiledb", "create"))
         if isinstance(create_entry, dict):
-            attrs: Tuple["attrs_.Attribute[Any]", ...] = cls.__attrs_attrs__
+            attrs: Tuple[Attribute, ...] = cls.__attrs_attrs__  # type: ignore[type-arg]
```

Comment:
since we already have `attrs_.Attribute` available here (the same object as `attr.Attribute`, the extra import can be avoided).